include/commissioner/commissioner.hpp

```diff
@@ -1147,6 +1161,36 @@ class Commissioner
      *         git repository.
      */
     static std::string GetVersion(void);
+
+    /**
+     * @brief Asynchronously query diagnostic decoded data from a Thread device.
+     *
+     * This method sends a DIAG_GET.qry message to the specified Thread device,
+     * requesting the set of diagnostic data indicated by `aDiagDataFlags`.
+     * The response, or any errors encountered, will be delivered to the provided `aHandler`.
+     *
+     * @param[in, out] aHandler        A handler to process the response or any errors.
+     *                                 This handler is guaranteed to be called.
+     * @param[in]      aAddr           Mesh local address of the target Thread device.
```

Comment:
Can the `aAddr` be a multicast address?

Reply:
so far this api only supports unicast address, I update the comments here, thanks.

Reply:
Does the spec support multicast address?